spec/unit/pops/evaluator/evaluating_parser_spec.rb

```diff
@@ -861,7 +861,8 @@
         # NOTE: these meta-esque parameters are not recognized as such
         "notify { id: message=>explicit} Notify[id][title]"   => /does not have a parameter called 'title'/,
         "notify { id: message=>explicit} Notify[id]['type']"   => /does not have a parameter called 'type'/,
-        "notify { id: message=>explicit } Notify[id]{message=>override}" => /'message' is already set on Notify\[id\]/
+        "notify { id: message=>explicit } Notify[id]{message=>override}" => /'message' is already set on Notify\[id\]/,
+        "notify { id: message => 'once', message => 'twice' }" => /'message' has already been set in this resource body/
```

Comment:
I think the reference to "body" here is not needed (I don't think users understand what a Resource Body is). If you feel it is important to make a distinction between the resource instance, and the PP language construct you could use the term "resource statement", or "resource declaration" (the last is the term used in the puppet documentation).